.gitignore

```diff
@@ -71,3 +71,7 @@ stickers/qr_codes
 
 # python deps
 stickers/src
+
+# package lock files
+package-lock.json
+yarn.lock
```

Comment:
Why do you want to add `yarn.lock` to `.gitignore`?

Reply:
While not required for managing smart contracts dependencies, I understand that `yarn.lock` is added in the witmon for some reason, so I will remove it from `.gitignore`.